met-api/src/met_api/services/engagement_custom_content_service.py

```diff
@@ -0,0 +1,53 @@
+"""Service for engagement custom content management."""
+from met_api.constants.membership_type import MembershipType
+from met_api.models.engagement_custom_content import EngagementCustom as EngagementCustomModel
+from met_api.services import authorization
+from met_api.utils.roles import Role
+
+
+class EngagementCustomContentService:
+    """Engagement custom content management service."""
+
+    @staticmethod
+    def get_custom_content(content_id):
+        """Get map by engagement custom content id."""
```

Comment:
What do you mean by "get map" here?

Reply:
Thanks for noticing this, I updated the comments here.